docs/src/architecture/accounts.md

```diff
@@ -76,11 +88,4 @@ The process is as follows:
 * Alice shares the new Account ID to Bob (eg. when Alice wants to receive funds)
 * Bob executes a transaction and creates a note that contains an asset for Alice
 * Alice consumes Bob's note to receive the asset in a transaction
-* Depending on the account storage mode (private vs. public) and transaction type (local vs. network) the Operator receives new Account ID eventually and - if transaction is correct - adds the ID to the Account DB
-
-## Account storage modes
-Account data - stored by the Miden Node - can be public, private, or encrypted. The third and fourth most significant bits of the account ID specifies whether the account data is public `00`, encrypted `01`, or private `11`.
-
-* Accounts with **public state**, where the actual state is stored onchain. These would be similar to how accounts work in public blockchains. Smart contracts that depend on public shared state should be stored public on Miden, e.g., DEX contract.
-* Account with **encrypted state**, where the account data is stored onchain but in encrypted text. It provides liveness guarantee of the protocol for the account in question.
-* Accounts with **private state**, where only the hash of the account is stored onchain. Users who want stay private and take care of their own data should choose this mode. The hash is defined as: `hash([account ID, 0, 0, nonce], [vault root], [storage root], [code root])`.
+* Depending on the account storage mode (private vs. public) and transaction type (local vs. network) the Operator receives the new Account ID eventually and - if the transaction is correct - adds the ID to the Account DB
```

Comment:
```suggestion
* Depending on the account storage mode (private vs. encrypted vs. public) and transaction type (local vs. network) the operator receives the new Account ID eventually and - if the transaction is correct - adds the ID to the Account DB
```

We say "depending on account storage mode and transaction type", but we never refer back to how this affects what the operator does. If we don't want to go in that amount of detail, I would just say the "operator adds the account data or just ID to the account DB" part